src/layers/XRLayer/XRLayer.js

```diff
@@ -12,13 +12,16 @@ import vs1 from './xr-layer-vertex.webgl1.glsl';
 import vs2 from './xr-layer-vertex.webgl2.glsl';
 import { lens, channels } from './shader-modules';
 import { padColorsAndSliders, getDtypeValues } from '../utils';
+import { INTERPOLATION_MODES } from '../../constants';
 
 const SHADER_MODULES = [
   { fs: fs1, fscmap: fsColormap1, vs: vs1 },
   { fs: fs2, fscmap: fsColormap2, vs: vs2 }
 ];
 
-function getRenderingAttrs(dtype, gl) {
+function getRenderingAttrs(dtype, gl, interpolation) {
+  const isLinear = interpolation === INTERPOLATION_MODES.LINEAR;
+  const filterSetting = isLinear ? GL.LINEAR : GL.NEAREST;
```

Comment:
Why not just let folks pass down `GL.LINEAR` or `GL.NEAREST` directly, instead of making our own string enum?

Or change ours to:

```javascript
export enum INTERPOLATION_MODES {  LINEAR = GL.LINEAR,  NEAREST = GL.NEAREST}
```

Reply:
Sounds good.